### DISCUSSION TITLE
Github Copilot prevents me from modifying variables while debugging in Visual Studio

### DISCUSSION THREAD

**Discussion by Beyondo** [community/community#32599] [Copilot Conversations]
When I try to modify a variable, let's say `m_width`

![image](https://user-images.githubusercontent.com/58893646/189502710-8f196a6d-c1aa-470e-9f0e-4b59bea8404c.png)

**The github copilot icon appears instead**

![image](https://user-images.githubusercontent.com/58893646/189502726-4190bfa6-2333-4164-93c3-5743ac557e94.png)

What's worse, even when I turn it off, the icon is still there

![image](https://user-images.githubusercontent.com/58893646/189502747-28fe2dda-e32f-4b41-a5de-b99709c26eef.png)

I've to delete the entire copilot extension for Visual Studio to work properly.

**Comment by iamcjwilliams** [comment]
Please try using Copilot 1.43.0.1 with VS 17.2.x which should not cause an issue.  Thanks!
